app/models/concerns/foreman/thread_session.rb

```diff
@@ -69,10 +69,15 @@ def current=(o)
             raise(ArgumentError, "Unable to set current User, expected class '#{self}', got #{o.inspect}")
           end
 
-          if o.is_a?(User)
-            Rails.logger.info "Current user: #{o.login} (#{o.admin? ? 'administrator' : 'regular user'})"
+          Rails.logger.debug do
```

Comment:
FYI: this line logs `[D|app|] <TrueClass> true`. However, even when updated, I'm not sure this PR actually fixes anything.

Reply:
Oh this was meant to be `debug? do`.